apps/condo/domains/meter/access/MeterReading.js

```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
         return await checkPermissionsInEmployedOrRelatedOrganizations(context, user, meterOrganization, 'canManageMeterReadings')
     }
 
+    if (operation === 'update' && itemId) {
+        const meterReading = await getByCondition('MeterReading', {
+            id: itemId,
+            deletedAt: null,
+        })
```

Comment:
What real case does this check cover? This looks like quite a major operation (extra query each reading before update)